Cargo.toml

```diff
@@ -5,7 +5,7 @@ members = [
     "parser",
 ]
 
-exclude = ["modules/test-module", "playground/web_bindings"]
+exclude = ["modules/test-module", "modules/table", "modules/html", "playground/web_bindings"]
```

Comment:
Do you think wildcards are allowed, so we can do `modules/*`? Would be nice so we don't miss adding a module

Reply:
Also a nice idea, tried it out but it does not seem to work. Can try a bit more once we refactor the modules directory.